### PR TITLE
Implement RoomAddrGetter

### PR DESCRIPTION
**Pull request by lftakakura**
Implement RoomAddrGetter to enable all Maestro functionalities using Minikube on development environment

### CODE REVIEW

File: api/app.go [hunk 1]
```diff
@@ -542,6 +545,14 @@ func (a *App) configureServer(showProfile bool) {
 	}
 }
 
+func (a *App) configureRoomAddrGetter() {
+	if a.Config.GetString("environment") != "development" {
```
Comment (Henrod) [inline api/app.go]: Save these strings as variables on constants file

File: models/room_address.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		return nil, maestroErrors.NewKubernetesError("no ports found", errors.New("no node port found to host room"))
+	}
+	return rAddresses, nil
+}
+
+// RoomAddressesFromNodePort is the struct that defines room addresses in development (using NodePort service)
+type RoomAddressesFromNodePort struct{}
+
+// Get gets room public addresses
+func (r *RoomAddressesFromNodePort) Get(room *Room, kubernetesClient kubernetes.Interface) (*RoomAddresses, error) {
```
Comment (Henrod) [inline models/room_address.go]: The two Get methods have a lot in common, maybe they can use a common private function

File: config/local.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+environment: "development"
```
Comment (Henrod) [inline config/local.yaml]: default value must be production

Comment (lftakakura) [reply config/local.yaml]: ok

**Pull request merged**